python/foglamp/plugins/south/cc2650async/cc2650async.py

```diff
@@ -165,148 +162,159 @@ def plugin_start(handle):
                 try:
                     pattern_index = tag.con.expect('Notification handle = .*? \r', timeout=4)
                 except pexpect.TIMEOUT:
-                    _LOGGER.error("SensorTagCC2650 async timeout")
-                    print("TIMEOUT exception!")
-                    break
-
-                # expect, if successful, will return the index of the pattern "Notification handle = " which ideally
+                    attempt_count += 1
+                    if attempt_count > 15:
+                        _LOGGER.error("SensorTagCC2650 {} async timeout")
+                        print("TIMEOUT!!")
```

Comment:
if this this for python3 -m, can we please put the same message as in logger error